python/triton/language/extra/libdevice.py

```diff
@@ -0,0 +1,10 @@
+from triton.runtime.driver import driver
```

Comment:
Why do we want to have HIP version of libdevice?

Reply:
Do you mean we should implement `mulhi` using the math library ?

Reply:
The idea is to expose some driver ops similar to what we do with cuda/libdevice